frontend/src/modules/entitieslist/components/Entity.js

```diff
@@ -83,21 +90,52 @@ export default class Entity extends React.Component<Props> {
         return 'missing';
     }
 
-    selectEntity = () => {
+    selectEntity = (e: SyntheticMouseEvent<HTMLLIElement>) => {
+        // Flow requires that we use `e.currentTarget` instead of `e.target`.
+        // However in this case, we do want to use that, so I'm ignoring all
+        // errors Flow throws there.
+        // $FLOW_IGNORE
```

Comment:
Is this comment actually accurate?

Reply:
I think so. It's a copy and paste, but Flow fails.

Reply:
Sorry, my question wasn't clear. I'm wondering if you can actually use `e.currentTarget` here instead of `e.target`?

Reply:
I tried that initially, but it always returns the `<li>` element.

Reply:
OK great! Just wanted to make sure it wasn't a wrong copy/paste. :-)